scripts/proposals/NomineeGovV2AIP/nominee-gov-v2-aip.json

```diff
@@ -0,0 +1,13 @@
roposes improvements to the ArbitrumDAO Constitution and the Security Council election process, which include:\n\n* Candidates are given a week to apply for a Security Council position before the nominee selection stage takes place.\n* Candidates need to sign a transaction from their EOA (Externally Owned Account) to apply for a Security Council position.\n* The ArbitrumDAO Constitution is updated to reflect these changes as well as correct previous mistakes in wording with regard to timing and quorum.\n\n## Motivation\n\nThis AIP aims to improve the election process ensuring that candidates are not overlooked during the nominee selection stage and that candidates stand an equal chance of being nominated to the following stage of the election process, irrespective of when in the registration period they express their interest. It also updates the system so contenders must provide a signed message before running in the elections - this ensures they are able to produce signed messages for the different chains that the Security Council needs the ability to act on.\n\nIssues with the current Security Council election process:\n\n* In the nominee Selection stage, because contender registration and endorsement occur in the same 7-day period, delegates are likely to overlook applications that are submitted at the end of the period. This results in candidates who submit their applications at the start of the period having an unfair advantage, as they are more likely to be endorsed and therefore progress to the next stage.\n\n* Candidates might sign up to the elections using an on-chain smart contract wallet which is not able to produce signatures valid for other ArbitrumDAO-governed chains.\n* The constitution does not clearly express the election procedure.\n\n## Specifications\n\nAccordingly, this AIP aims to implement the following changes:\n\n1. A 7-day ‘Contender Submission’ stage before the ‘Nominee Selection’ stage. Accordingly, the duration of the entire election process will increase from 42 days to 49 days.\n2. For candidates to apply for a Security Council position, they are required to provide a signed message from their wallet.\n3. The wording on the ArbitrumDAO Constitution is updated to correctly reflect how quorum for proposals (as well as the election process) is handled, which should also take into account ‘abstain’ votes.\n4. The wording on the ArbitrumDAO Constitution is updated to reflect the new format of the Security Council elections.\n\nThe code representing the proposed changes is final and available for full review in: [https://github.com/ArbitrumFoundation/governance/pull/231](https://github.com/ArbitrumFoundation/governance/pull/231) and [https://github.com/ArbitrumFoundation/docs/pull/731](https://github.com/ArbitrumFoundation/docs/pull/731)\n\nThe Governance Action contract was deployed and is available for review in https://arbiscan.io/address/0xd9a2e0e5d7509f0bf1b2d33884f8c1b4d4490879#code\n\n-------------------------\n",
```

Comment:
this will get updated if we remove the heading and footer that I suggested above